.github/workflows/slash-commands.yml

```diff
@@ -1,37 +0,0 @@
-name: Slash Commands
-
-on:
-  issue_comment:
-    types: [created]
-
-jobs:
-  dispatch:
-    runs-on: ubuntu-latest
-    if: contains(github.event.comment.html_url, '/pull/')
-
-    steps:
-      - name: Get branch name based on PR ID
-        uses: andrevalentin/get-branch-name-by-pr@v1
-        id: getBranchName
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          pr-id: ${{ github.event.issue.number }}
```

Comment:
We had to find the PR's branch in order for the Cypress worfklow to be triggered on the right reference branch. Since we don't trigger a separate workflow, this action is no longer required.